content/guides/java/tracers.md

```diff
@@ -2,7 +2,67 @@
 title: "Java: Tracers"
 ---
 
-* Setting up your tracer
-* Using the Global Tracer
-* Accessing the Active Span
```

Comment:
I guess it's missing this item: "Accessing the Active Span"

Reply:
I covered that in the `Scopes` section (scopes.md, that is)